src/test/java/org/apache/maven/plugins/dependency/TestGetMojo.java

```diff
@@ -96,6 +117,27 @@ public void testRemoteRepositories()
         mojo.execute();
     }
 
+    /**
+     * Test remote repositories parameter with basic authentication
+     *
+     * @throws Exception in case of errors
+     */
+    public void testRemoteRepositoriesAuthentication()
+        throws Exception
+    {
+        org.eclipse.jetty.server.Server server = createServer( 8000 );
+        server.start();
+
+        setVariableValueToObject( mojo, "remoteRepositories", "myserver::default::http://localhost:8000/maven" );
+        mojo.setGroupId( "test" );
+        mojo.setArtifactId( "test" );
+        mojo.setVersion( "1.0" );
+
+        mojo.execute();
+
+        server.stop();
```

Comment:
Pls call it within the `finally` block. This is a typical problem of integration tests that they failed and the server is never stopped.

Reply:
I've added the server.stop() to a finally block.